frontend/src/components/analystTools/GCDocumentsComparisonTool.js

```diff
@@ -570,433 +570,430 @@ const GCDocumentsComparisonTool = (props) => {
 	};
 	
 	return (
-		<>
-			<Grid container style={{marginTop: 20}}>
-				<Grid item xs={12}>
-					<div  style={{display: 'flex'}}>
-						<div style={{fontWeight:'bold', alignItems: 'center', fontFamily: 'Noto Sans',}}>
-					The Document Comparison Tool enables you to input text and locate policies in the GAMECHANGER policy repository with semantically similar language. Using the Document Comparison Tool below, you can conduct deeper policy analysis and understand how one piece of policy compares to the GAMECHANGER policy repository.
-						</div>
-						{(!loading && returnedDocs.length > 0) &&
-							<div style={{display: 'flex', marginLeft: 20}}>
-								<FormControl variant="outlined" classes={{root:classes.root}} style={{marginLeft: 'auto', margin: '-10px 0px 0px 0px', minWidth: 195}}>
-									<InputLabel classes={{root: classes.formlabel}} id="view-name-select">Sort</InputLabel>
-									<Select
-										className={`MuiInputBase-root`}
-										labelId="re-view-name"
-										label="View"
-										id="re-view-name-select"
-										value={sortType}
-										onChange={(event) => {
-											setSortType(event.target.value);
-											setNeedsSort(true);
+		<Grid container style={{ marginTop: 20, paddingBottom: 20 }}>
```

Comment:
The diff is confused here which makes this seem like a lot is changing, but the only changes are adding `paddingBottom: 20` to the container style, and removing the unnecessary encapsulating fragment `<>...</>` as there is already only a single Grid element being returned.